<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bivariate Plot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="centralwidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Graph</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QRadioButton" name="scatterradioButton">
               <property name="text">
                <string>Scatter</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="ellipseradioButton">
               <property name="text">
                <string>Ellipse</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="boxcoxradioButton">
               <property name="text">
                <string>Box Cox</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="histogramradioButton">
               <property name="text">
                <string>bi Histogram</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Variables</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="Xlabel">
               <property name="text">
                <string>X axis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="XcomboBox">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="Ylabel">
               <property name="text">
                <string>Y axis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="YcomboBox">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>Labelling</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Title</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="TlineEdit">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>X axis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="XlineEdit">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Y axis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="YlineEdit">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="MarkergroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Marker</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QCheckBox" name="PcheckBox">
               <property name="text">
                <string>Points</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="XcheckBox">
               <property name="text">
                <string>X Label</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QCheckBox" name="LcheckBox">
               <property name="text">
                <string>Lines</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="YcheckBox">
               <property name="text">
                <string>Y Label</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QCheckBox" name="XGcheckBox">
               <property name="text">
                <string>X Grid</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="XMcheckBox">
               <property name="text">
                <string>X Marks</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QCheckBox" name="YGcheckBox">
               <property name="text">
                <string>Y Grid</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="YMcheckBox">
               <property name="text">
                <string>Y Marks</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QCheckBox" name="CcheckBox">
               <property name="text">
                <string>Colors</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="VcheckBox">
               <property name="text">
                <string>Values</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>n bins</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox">
               <property name="minimum">
                <number>5</number>
               </property>
               <property name="maximum">
                <number>200</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="ApplyButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Apply</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ResetButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="mplwindow" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="mplvl"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>648</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>764</x>
     <y>705</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>717</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
